Doc/library/enum.rst

```diff
@@ -267,6 +267,11 @@ Data Types
          appropriate value will be chosen for you.  See :class:`auto` for the
          details.
 
+         While mutable/unhashable values, such as :class:`dict` or :class:`list`, can
```

Comment:
I'd prefer to see the two commas on this line discarded, they introduce too many pauses.

Reply:
Pauses aren't necessarily bad, and I don't think parentheses are warranted here.